src/ubuntu/18.04/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-amd64
+
+USER root
+
+# Install additional Mono build dependencies
+RUN apt-get update \
+    && apt-get install -qq -y \
+        autoconf \
+        libtool \
+        automake \
+        build-essential \
+        gettext \
+        wget \
+        bc \
```

Comment:
Please list these in alphabetical order.  Same applies to the other Dockerfiles.